### PR TITLE
MCP-1787:  increase rabbit timeout

### PR DESCRIPTION
**Pull request by dimitri-amida**
# Feature Name 

## Description

Requests time out

increase timeout from 20 to 60 seconds

### Jira Tickets

- [MCP-1787](https://amida.atlassian.net/browse/MCP-1787)

## How to test this PR

- Repro steps in ticket

## Reminders

- Review [Pull-Requests](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests) guidelines
- [ ] If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) and
[Roadmap](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Roadmap) wiki pages
- [ ] If changing software behavior, post a link to the PR in Slack channel #benefits-virtual-regional-office

### CODE REVIEW

File: api/src/main/java/gov/va/vro/api/model/ClaimProcessingException.java [hunk 1]
```diff
@@ -1,14 +1,37 @@
...
+  }
+
+  public String getOriginalMessage() {
+    if (exception == null) {
+      return message;
+    }
+    if (exception.getCause() == null) {
+      return exception.getMessage();
+    }
+    return exception.getCause().getMessage();
```
Comment (dimitri-amida) [inline api/src/main/java/gov/va/vro/api/model/ClaimProcessingException.java:35]: Get the real cause if available

File: service/provider/src/main/java/gov/va/vro/service/provider/camel/CamelConfiguration.java [hunk 2]
```diff
@@ -89,22 +80,4 @@ ConnectionFactory rabbitmqConnectionFactory() {
     factory.setPassword(messageQueueProps.getPassword());
     return factory;
   }
-
-  public static String servletName = "VroCamelRestServlet";
```
Comment (dimitri-amida) [inline service/provider/src/main/java/gov/va/vro/service/provider/camel/CamelConfiguration.java:93]: We are not using camel REST

File: service/provider/src/main/java/gov/va/vro/service/provider/camel/PrimaryRoutes.java [hunk 3]
```diff
@@ -59,8 +59,7 @@ private void configureRouteClaimSubmit() {
         // Use Properties not Headers
         // https://examples.javacodegeeks.com/apache-camel-headers-vs-properties-example/
         .setProperty("diagnosticCode", simple("${body.diagnosticCode}"))
-        .routingSlip(method(SlipClaimSubmitRouter.class, "routeClaimSubmit"))
-        .log(">>5> ${body}");
```
Comment (dimitri-amida) [inline service/provider/src/main/java/gov/va/vro/service/provider/camel/PrimaryRoutes.java:63]: log message is too big and contains potential PII

**Review by ianwolson — APPROVED** [review]

**Review by wlin2019 — APPROVED** [review]
Left some comments. LGTM

**Comment by dimitri-amida** [comment]
Thanks @wlin2019 ! I addressed your comments. Please re-approve when you get a chance.

**Review by ianwolson — APPROVED** [review]

File: api/src/main/java/gov/va/vro/api/model/ClaimProcessingException.java [hunk 4]
```diff
@@ -1,14 +1,34 @@
...
+  }
+
+  public ClaimProcessingException(
+      String claimSubmissionId, HttpStatus httpStatus, Exception exception) {
+    super(exception);
+    this.claimSubmissionId = claimSubmissionId;
+    this.httpStatus = httpStatus;
+  }
+
+  public String getOriginalMessage() {
```
Comment (wlin2019) [inline api/src/main/java/gov/va/vro/api/model/ClaimProcessingException.java:24]: It is not really "original" message, is it? Could the code be a bit simpler? But it is okay if there is a use for it.

**Review by wlin2019 — APPROVED** [review]
I don't know Camel code. LGTM

File: api/src/main/java/gov/va/vro/api/model/ClaimProcessingException.java (hunk 4 above)
Comment (dimitri-amida) [reply api/src/main/java/gov/va/vro/api/model/ClaimProcessingException.java:24]: The message that was displayed before was a generic exception saying that something occured in exchange XYZ. Going one level down in the message sequence reveals the actual reason for the exception, which in this case was a timeout. Perhaps in the future I will store the entire stacktrace as string if necessary.

**Pull request merged**